packages/react-storage/src/components/StorageBrowser/actions/handlers/types.ts

```diff
@@ -1,10 +1,9 @@
-import {
-  LocationCredentialsProvider,
-  Permission,
-} from '../../storage-internal';
+import { LocationCredentialsProvider } from '../../storage-internal';
 
 export type LocationType = 'OBJECT' | 'PREFIX' | 'BUCKET';
 
+export type LocationPermissions = ('delete' | 'get' | 'list' | 'write')[];
```

Comment:
nit: Should we just keep this `Permissions` from `Permission` rather than prefixing with `Location`?

Reply:
`LocationPermissions` always co-exists with `LocationType`. It makes sense for both of them having the `Location` prefix